lib/src/room.dart

```diff
@@ -51,16 +51,32 @@ abstract class Room implements Listenable {
   Stream<Message> get messageStream;
 
   /// Joins the room with the provided local audio and video stream.
-  ///
-  /// If the room should be joined with the audio and/or video muted, disable the corresponding track(s) before calling
-  /// [join].
   Future<void> join(MediaStream localStream);
 
-  /// Mutes or un-mutes the local audio stream.
-  void setAudioMuted(bool muted);
+  /// Whether the local audio is muted.
+  bool get isAudioMuted;
+
+  /// Mutes or un-mutes the local audio.
+  Future<void> setAudioMuted(bool muted);
+
+  /// Whether the local video is muted.
+  bool get isVideoMuted;
+
+  /// Mutes or un-mutes the local video.
+  Future<void> setVideoMuted(bool muted);
 
-  /// Mutes or un-mutes the local video stream.
-  void setVideoMuted(bool muted);
+  /// Whether the presentation is muted.
+  bool get isPresentationMuted;
+
+  /// Mutes or un-mutes the presentation.
+  Future<void> setPresentationMuted(bool muted);
+
+  /// Enables or disables privacy mode.
+  ///
+  /// In privacy mode, the local audio, video and presentation are _all_ muted. This is equivalent to calling
+  /// [setAudioMuted], [setVideoMuted] and [setPresentationMuted], and is intended as a convenience for apps that do not
+  /// support muting the tracks independently.
+  Future<void> setPrivacyMode(bool enabled);
```

Comment:
This is for mobile or glasses apps that have a single "privacy mode" toggle. This allows us to call `_updateParticipantStatus` once instead of once per mute.